### PR TITLE
fix toml property for recaptcha

### PR DESCRIPTION
**Pull request by wijith7**
fix - https://github.com/wso2/product-is/issues/6131

### CODE REVIEW

File: features/org.wso2.carbon.identity.captcha.server.feature/resources/conf/captcha-config.properties.j2 [hunk 1]
```diff
@@ -33,5 +33,8 @@ recaptcha.site.key={{identity.recaptcha.site_key}}
 # reCaptcha secret key
 recaptcha.secret.key={{identity.recaptcha.secret_key}}
```
Comment (hasinidilanka) [inline features/org.wso2.carbon.identity.captcha.server.feature/resources/conf/captcha-config.properties.j2]: Can we remove the "identity" prefix from all the keys relate to recaptcha. 
Eg: identity.recaptcha.secret_key to recaptcha.secret_key

File: features/org.wso2.carbon.identity.captcha.server.feature/resources/conf/captcha-config.properties.j2 [hunk 2]
```diff
@@ -19,19 +19,25 @@
...
+recaptcha.secret.key={{recaptcha.secret_key}}
+
+# login.do URL paths
+{% if recaptcha.redirect_urls is defined %}
+recaptcha.failed.redirect.urls={{recaptcha.redirect_urls}}
+{% endif %}
 
 # reCaptcha failed redirect urls
 #recaptcha.failed.redirect.urls=
+[]
```
Comment (Kanapriya) [inline features/org.wso2.carbon.identity.captcha.server.feature/resources/conf/captcha-config.properties.j2]: Add new line here

File: features/org.wso2.carbon.identity.captcha.server.feature/resources/conf/captcha-config.properties.j2 (hunk 1 above)
Comment (wijith7) [reply features/org.wso2.carbon.identity.captcha.server.feature/resources/conf/captcha-config.properties.j2]: fixed

File: features/org.wso2.carbon.identity.captcha.server.feature/resources/conf/captcha-config.properties.j2 (hunk 2 above)
Comment (wijith7) [reply features/org.wso2.carbon.identity.captcha.server.feature/resources/conf/captcha-config.properties.j2]: fix with - https://github.com/wso2-extensions/identity-governance/pull/326

**Pull request merged**